components/cli/vendor/github.com/docker/docker/client/errors.go

```diff
@@ -228,7 +228,7 @@ func IsErrPluginPermissionDenied(err error) bool {
 // NewVersionError returns an error if the APIVersion required
 // if less than the current supported version
 func (cli *Client) NewVersionError(APIrequired, feature string) error {
-	if versions.LessThan(cli.version, APIrequired) {
+	if cli.version != "" && versions.LessThan(cli.version, APIrequired) {
```

Comment:
Brought in by PR:
* https://github.com/moby/moby/pull/33887 Fix NewVersionError() for clients using default version